src/pages/edge/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -8,70 +8,88 @@ keywords:
 
 # Release notes
 
+## March 11, 2025
+
+### Adobe Journey Optimizer - Decisioning extension 1.1.0
+
+Added configuration capability to the Adobe Journey Optimizer - Decisioning extension in the Tags UI for mobile Tag Properties.
+
+Users can now configure:
+
+* Request timeout - Set the amount of time (in seconds) to wait for a response from the decisioning services before timing out.
+
+### Android Optimize 3.3.1
+
+* Added support for respecting and handling the request timeout configuration set in the Adobe Journey Optimizer - Decisioning extension within the Tags UI.
+
+### iOS Optimize 5.3.0
+
+* Added support for respecting and handling the request timeout configuration set in the Adobe Journey Optimizer - Decisioning extension within the Tags UI.
+
 ## February 17, 2025
 
 ### React Native Optimize 6.1.0
 
-* **[Breaking Change]** Updated Offer Score data type of  from `Integer` to `Double` .
+* **[Breaking Change]** Updated Offer Score data type of  from `Integer` to `Double` .
 * To prevent build issues in **6.0.x**, upgrade `@adobe/react-native-aepoptimize` to **6.1.0 or later**.
 
 ### Android Optimize 3.3.0
 
-* **[Breaking Change]** Updated Offer Score data type of from `Integer` to `Double`.
-* Optimize SDK only listens to preview events if it contains `debug` object with `eventType` as `com.adobe.eventType.edge` and `eventSource` as `personalization:decisions`.
+* **[Breaking Change]** Updated Offer Score data type of from `Integer` to `Double`.
```

Comment:
Same comment as above - it might be clearer if we just make this "Updated Offer Score data type from `Integer` to `Double`.